Linux/Car_Client/chronoscomm.cpp

```diff
@@ -20,6 +20,8 @@
 #include <cmath>
 
 #define PIN_OUT 4
+#define HEARTBEAT_TIME_MS 1000
```

Comment:
```suggestion
#define MAX_HEAB_PERIOD_MS 1000
```

Reply:
One second is a long time when it comes to an emergency. I know that the RC car most likely will not kill anyone, but if we make it lax to errors in Maestro then we will never fix those errors.

Reply:
Agreed. Have we defined any timeout periods? 100 ms or perhaps even lower?

Reply:
If I run a test on the RC car with 100 ms I occasionally get a timeout between 101 and 150 ms (using a print out in ChronosComm::checkLastHeabRestart()). Not sure if there is some kind of skit bakom spakarna or some stuff going on in CarClient that takes time or what the problem is, because as far as I can see the HEAB is sent approx every 10 ms (using Wireshark).

Reply:
Hmmm, I did not notice that but then again I was using 1000 ms and might have missed/disregarded the extra ~50 ms. This might be worth looking into, 50 ms is not a lot but it would be good to know what is causing it. 

It might be worth mentioning as well that we had a discussion before which you should perhaps have been included in with in regards to the timeout for HEAB since you use a similar approach for RCMM. 

We were thinking that a better approach than just measuring a timed delay would be to implement some sort of calculation which takes into account the current speed and the mass of the vehicle. In essence making it so that that we can miss fewer messages at a high speed, and be more lenient when it comes to missing messages at slower speeds.

Reply:
Ok, good to know. That makes sense. 
Worth pointing out is that I have the same problem with timer timeouts for rcmm. Not sure how to go forward with this (except for increasing the timeout!) :)